pocld/shared_cl_context.cc

```diff
@@ -1905,6 +1901,13 @@ int SharedCLContext::buildOrLinkProgram(
 
     for (cl_uint arg_index = 0; arg_index < temp_kernel.num_args; ++arg_index) {
       ArgumentInfo_t &temp_arg = program->kernel_meta[i].arg_meta[arg_index];
+      struct {
+        bool AccessQualifier;
+        bool AddressQualifier;
+        bool TypeQualifier;
+        bool ArgType;
+        bool ArgName;
+      } Have{};
```

Comment:
What's the idea for the temporary struct here?

Reply:
I thought it might help with keeping track of them but idk. Turned them into standalone variables now.